src/drive/targets/services/dacc.js

```diff
@@ -35,11 +35,17 @@ export const run = async () => {
     excludedSlug,
     nonExcludedGroupLabel,
     measureName,
-    remoteDoctype
+    remoteDoctype,
+    maxFileDateQuery
   } = daccFileSizeFlag
-  const aggregationDate = endOfMonth(subMonths(new Date(), 1))
+
+  const aggregationDate =
+    maxFileDateQuery || endOfMonth(subMonths(new Date(), 1))
 
   const sizesBySlug = await aggregateFilesSize(client, aggregationDate)
+  if (Object.keys(sizesBySlug).length < 1) {
+    log('info', `No files found to aggregate with date ${aggregationDate}`)
```

Comment:
I would expect 2 tests for this kind of PRs, but I wonder if there are drawbacks testing service. I asked a question to @Crash-- , and according to his answer, I will approve or suggest the 2 tests

Reply:
=> https://github.com/cozy/cozy-drive/commit/ced3151db01869145235133e2a1bd74b972c0918

you can cherry pick this commit (and modify to comply with the business need) :-)

Reply:
That's very nice, I slightly adapted it with [25d28d6](https://github.com/cozy/cozy-drive/pull/2499/commits/25d28d6b355233d0315a1029ca674df006785f8d) :)